File: resQfood-user_management/src/main/java/models/fatma/Participations.java
package models.fatma;

public class Participations {
    private  int idP ;
    private int id_event ;
    private String particip_name ;

    public String getParticip_name() {
        return particip_name;
    }

    public void setParticip_name(String particip_name) {
        this.particip_name = particip_name;
    }

    public Participations(int idP, int id_event) {
        this.idP = idP;
        this.id_event = id_event;
    }
    public Participations(int idP, int id_event, String particip_name) {
        this.idP = idP;
        this.id_event = id_event;
        this.particip_name = particip_name;
    }


    public Participations(int id_event) {
        this.id_event = id_event;
    }
    public Participations(){

    }

    public int getIdP() {
        return idP;
    }

    public void setIdP(int idP) {
        this.idP = idP;
    }

    public int getId_event() {
        return id_event;
    }

    @Override
    public String toString() {
        return "Participations{" +
                "idP=" + idP +
                ", id_event=" + id_event +
                ", particip_name=" + particip_name +
                '}';
    }

    public void setId_event(int id_event) {
        this.id_event = id_event;
    }
}

File: src/Entity/Participations.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Participations
 *
 * @ORM\Table(name="participations", indexes={@ORM\Index(name="fk_ide", columns={"id_event"})})
 * @ORM\Entity
 */
class Participations
{
    /**
     * @var int
     *
     * @ORM\Column(name="idP", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idp;

    /**
     * @var int
     *
     * @ORM\Column(name="id_event", type="integer", nullable=false)
     */
    private $idEvent;

    /**
     * @var string
     *
     * @ORM\Column(name="particip_name", type="string", length=200, nullable=false)
     */
    private $participName;

    public function getIdp(): ?int
    {
        return $this->idp;
    }

    public function getIdEvent(): ?int
    {
        return $this->idEvent;
    }

    public function setIdEvent(int $idEvent): static
    {
        $this->idEvent = $idEvent;

        return $this;
    }

    public function getParticipName(): ?string
    {
        return $this->participName;
    }

    public function setParticipName(string $participName): static
    {
        $this->participName = $participName;

        return $this;
    }


}
